checkpoint_1/Readme.md

```diff
@@ -1,50 +1,49 @@
 
 Create a directory named foo_bar
 
-    Please use markdown formatting listing the command(s) you ran to accomplish this
+   `mkdir foo-bar`
```

Comment:
Attention to detail... You were asked to create `foo_bar`, you created `foo-bar`...  👮